### PR TITLE
Improve values

### PR DESCRIPTION
**Pull request by tuxtof**
- remove default for some required values
- improve values description
- enforce required values declaration

### CODE REVIEW

File: charts/nutanix-csi-storage/templates/ntnx-sc.yaml [hunk 1]
```diff
@@ -42,8 +45,12 @@ parameters:
   csi.storage.k8s.io/snapshotter-secret-namespace: {{ .Release.Namespace }}
 deletionPolicy: Delete
 {{- end }}
----
+
 {{- if eq .Values.fileClass true }}
+---
```
Comment (sanchitatntnx) [inline charts/nutanix-csi-storage/templates/ntnx-sc.yaml:50]: Why this change ?

File: charts/nutanix-csi-storage/templates/ntnx-sc.yaml [hunk 2]
```diff
@@ -59,8 +66,11 @@ parameters:
     nfsPath: {{ .Values.filePath }}
 reclaimPolicy: {{ .Values.fileClassRetention }}
 {{- end }}
----
+
 {{- if eq .Values.dynamicFileClass true }}
+---
```
Comment (sanchitatntnx) [inline charts/nutanix-csi-storage/templates/ntnx-sc.yaml:71]: Again, why this change ( shifting --- below if statement) ?

**Review by sanchitatntnx — APPROVED** [review]
Please address some comments before merging. Looks good otherwise.

File: charts/nutanix-csi-storage/templates/ntnx-sc.yaml (hunk 2 above)
Comment (tuxtof) [reply charts/nutanix-csi-storage/templates/ntnx-sc.yaml:71]: it was creating
---
---
in rendered file depending of SC choice

**Review by subodh01 — APPROVED** [review]
